changes/unreleased/4750.jJBu7iAgZa96hdqcpHK96W.toml

```
@@ -0,0 +1,35 @@
+features = "Use `timedelta` to represent time periods in classes"
+deprecations = """In this release, we're migrating attributes that represent durations/time periods from :ob:`int` type to Python's native :class:`datetime.timedelta`.
+
+Enable the ``PTB_TIMEDELTA`` environment variable to adopt :obj:`timedelta` now. Support for :obj:`int` values is deprecated and will be removed in a future major version.
```

```suggestion
Set the ``PTB_TIMEDELTA=1`` environment variable to adopt :obj:`timedelta` now. Support for :obj:`int` values is deprecated and will be removed in a future major version.
```
it's not clear to me what happens when you set the env variable, are timedelta objects returned? If that's the case, the release note should mention that.